Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -87,6 +87,34 @@ lemma order_eq_int_iff {n : ℤ} (hf : MeromorphicAt f x) : hf.order = n ↔
     exact ⟨fun h ↦ ⟨g, hg_an, hg_ne, h ▸ hg_eq⟩,
       AnalyticAt.unique_eventuallyEq_zpow_smul_nonzero ⟨g, hg_an, hg_ne, hg_eq⟩⟩
 
+lemma order_neg_iff_tendsto_cobounded (hf : MeromorphicAt f x) :
```

Comment:
I believe this lemma will be useful in a variety of settings, but as a relative newcomer to lean, I had to sit down and think what it actually says. Personally, I would greatly appreciate a docstring to help the reader.

Reply:
Hopefully all this is already better in the current version, with docstrings and everything.

Reply:
Agreed.